libraries/rush-lib/src/api/Rush.ts

```diff
@@ -103,7 +103,7 @@ export class Rush {
     options = Rush._normalizeLaunchOptions(options);
 
     Rush._assignRushInvokedFolder();
-    RushXCommandLine._launchRushXInternal(launcherVersion, { ...options });
+    RushXCommandLine.launchRushX(options.isManaged);
```

Comment:
@nebosite This is not correct.  The `launchRushX()` is the entry point for the `rushx` CLI binary (via the [VersionSelector  system](https://github.com/microsoft/rushstack/blob/d95ad804d7abaf67c707513dd50e3c6672ab546c/apps/rush/src/RushCommandSelector.ts#L60)).  The options such as `alreadyReportedNodeTooNewError` are important features. We cannot simply discard them here.

Reply:
Ah, I see.  I didn't think about the API.  I'll fix it tongith.